### PR TITLE
Monorepo

### PR DESCRIPTION
**Pull request by kiwiupover**
Move ember-cli-fastboot addon to packages/ember-cli-fastboot

This PR is the base for moving tests to a monorepo structure following the discussion here. 

https://github.com/ember-fastboot/ember-cli-fastboot/issues/765

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+name: CI
```
Comment (rwjblue) [inline .github/workflows/ci.yml:1]: Should drop away when we rebase

File: .travis.yml [hunk 2]
```diff
@@ -29,6 +29,7 @@ cache:
 before_install:
   - npm config set spin false
   - npm install -g npm@4
+  - cd packages/ember-cli-fastboot
```
Comment (rwjblue) [inline .travis.yml:32]: I think this change will have to be made over in the GH actions setup.

File: appveyor.yml [hunk 3]
```diff
@@ -17,6 +17,7 @@ branches:
 install:
   - ps: Install-Product node $env:nodejs_version
   - npm config set spin false
+  - cd packages/ember-cli-fastboot
```
Comment (rwjblue) [inline appveyor.yml:20]: I think this file will be deleted

File: packages/ember-cli-fastboot/package.json [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        "infile": "CHANGELOG.md"
+      }
+    },
+    "git": {
+      "tagName": "v${version}"
+    },
+    "github": {
+      "release": true
+    }
+  },
```
Comment (rwjblue) [inline packages/ember-cli-fastboot/package.json:106]: We'll need to tweak this too, I'll do that in a follow up.

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Yarn Install
         run: yarn install --ignore-engines
       - name: Run Mocha Tests
-        run: yarn test:mocha
+        run: yarn workspace ember-cli-fastboot test:mocha
```
Comment (rwjblue) [inline .github/workflows/ci.yml:44]: These commands are failing CI, not exactly sure why. Can you run it locally without error?

Comment (kiwiupover) [reply .github/workflows/ci.yml:44]: I closed the PR and re opened the PR again and it is now working.

@rwjblue ^^

**Pull request merged**